pkg/runctl/controller_test.go

```diff
@@ -135,8 +135,8 @@ func Test_Controller_syncHandler_givesUp_onPipelineRunNotFound(t *testing.T) {
 		Return(nil, nil)
 
 	// EXERCISE
-	examinee := NewController(cf, mockPipelineRunFetcher, metrics.NewMetrics())
-
+	examinee := NewController(cf, metrics.NewMetrics())
+	examinee.pipelineRunFetcher = mockPipelineRunFetcher
 	// VERIFY
 	assert.NilError(t, examinee.syncHandler("foo/bar"))
```

Comment:
The call to `syncHandler` is the actual exercise. The `NilError` check is verification. Please separate those.